godtools/Managers/DownloadResources/DownloadedResourceManager.swift

```diff
@@ -57,12 +58,13 @@ class DownloadedResourceManager: GTDataManager {
                 cachedResource.code = remoteResource.abbreviation
                 cachedResource.name = remoteResource.name
                 
-                let remoteTranslations = remoteResource.translations!
+                let remoteTranslations = remoteResource.latestTranslations!
                 for remoteTranslationGeneric in remoteTranslations {
                     let remoteTranslation = remoteTranslationGeneric as! TranslationResource
                     
                     let cachedTranslation = Translation.mr_findFirstOrCreate(byAttribute: "remoteId", withValue: remoteTranslation.id!, in: context)
                     
+                    cachedTranslation.language = Language.mr_findFirst(byAttribute: "remoteId", withValue: remoteTranslation.language?.id ?? "-1", in: context)
```

Comment:
Would you please set the value to be searched on a variable and here just use the variable?